content/node/authors/index.html.haml

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+section: blog
+title: Jenkins Community Blog Contributors
```

Comment:
I would not say that all authors are actually blog contributors, but IMHO it is ok

Reply:
@oleg-nenashev do you have an alternative suggestion for the title? Currently the author pages are generated only for author names/ids extracted from blog posts.